### PR TITLE
Add a few more examples to commands with -—help

### LINKED ISSUE #2485: `skaffold build --help` should show example of using -o

### PR DESCRIPTION
**Pull request by dgageot**
Fix #2485

Signed-off-by: David Gageot <david@gageot.net>

### CODE REVIEW

File: cmd/skaffold/app/cmd/build.go [hunk 1]
```diff
@@ -41,6 +41,11 @@ var (
 func NewCmdBuild() *cobra.Command {
 	return NewCmd("build").
 		WithDescription("Build the artifacts").
+		WithLongDescription("Build, test and tag the artifacts").
+		WithExample("Build all the artifacts", "build").
+		WithExample("Build artifacts with a profile activated", "build -p <profile>").
+		WithExample("Build artifacts whose image name contain <db>", "build -b <db>").
```
Comment (balopat) [inline cmd/skaffold/app/cmd/build.go]:
```suggestion
		WithExample("Build artifacts whose image name contains <db>", "build -b <db>").
```

File: cmd/skaffold/app/cmd/build.go [hunk 2]
```diff
@@ -41,6 +41,11 @@ var (
...
+		WithExample("Quietly build artifacts and output the image names as json", "build -q -o '{{json .}}'").
```
Comment (balopat) [inline cmd/skaffold/app/cmd/build.go]: you don't need the `-o '{{json .}}'`, by default -q spits out the json on stdout...

Comment (balopat) [reply cmd/skaffold/app/cmd/build.go]: How about ` skaffold build -q > build_result.json`
+ one more example candidate: `skaffold build -q > skaffold deploy`

**Comment by dgageot** [comment]
@balopat Thanks for the review! I've taken your feedback into account

**Pull request merged**